pwndbg/vmmap.py

```diff
@@ -34,6 +34,7 @@
 # List of manually-explored pages which were discovered
 # by analyzing the stack or register context.
 explored_pages = []
+custom_pages = []
```

Comment:
This looks as if the upper comment is also about `custom_pages`, please add:
`# List of custom pages that can be managed manually by vmmap_* commands family`

Reply:
Added